docs/admin/index.rst

```diff
@@ -122,6 +122,42 @@ For instance, make a user admin on GitLab.
 3. modify any users you want to modify (e.g. to make them admin)
 4. turn the automatic redirect back on
 
+Enabling notebooks for anonymous users
+--------------------------------------
+
+Interactive sessions launched from public notebooks can be made available to logged-out users. However,
+this feature is deactivated by default. In order to activate it, you need a secondary deployment of the
+renku-notebooks chart (including a secondary Jupyterhub as a dependency) to handle the anonymous
+("temporary") sessions.
+
+1. Set :code:`global.anonymousSessions.enabled: true` in your values.yaml file. If you are
+   updating an existing deployment you have to set :code:`global.anonymousSessions.postgresPassword.overwriteOnHelmUpgrade: true`
+   for the upgrade and then back to `false` for upcoming deployments.
```

Comment:
I would also mention something about `anonymousSessions.autoCreateTmpNamespace`, otherwise it's undocumented

Reply:
It's something that is only necessary because of our dev cluster setup with Rancher and it's documented in `values.yaml`, so I'd prefer to leave it the way it is.

Reply:
I made the comment in the `values.yaml` file a bit more clear.